zkevm-circuits/src/evm_circuit/execution/shl_shr.rs

```diff
@@ -153,8 +162,13 @@ impl<F: Field> ExecutionGadget<F> for ShlShrGadget<F> {
         self.same_context.assign_exec_step(region, offset, step)?;
         let indices = [step.rw_indices[0], step.rw_indices[1], step.rw_indices[2]];
         let [pop1, pop2, push] = indices.map(|idx| block.rws[idx].stack_value());
-        let shf0 = pop1.to_le_bytes()[0];
-        let divisor = if U256::from(shf0) == pop1 {
+        let shf0 = u128::from(pop1.to_le_bytes()[0]);
+        let shf_lt256 = pop1
```

Comment:
Nitpick: The maximum value of the sum of all bytes in a word is `0x1fe0` (`hex(0xff * 32)`) so I think this should fit in `u64`.  Is there any particular reason to use `u128`?

Reply:
Fixed `shf0` and `shf_lt256` to type u64, and replaced `+` with `checked_add` for shf_lt256 calculation in commit [b621563](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1151/commits/b6215631936a1b1d792ba29ea9235ebdb60d942e).